astro/src/content/docs/_shared/_passwordless-login-templates.mdx

```diff
@@ -0,0 +1,8 @@
+import FreemarkerTemplate from 'src/components/FreemarkerTemplate.astro';
```

Comment:
Rather than our previous setup, I thought it made more sense to read the sample ftl/template files off the local disk rather than fetch from GitHub, on a hard coded branch name, for a file in this repo.

Reply:
@andrewpai see the old templates i dropped. and I didn't replace the templates themselves, just how we render them here.